main.go

```diff
@@ -123,7 +127,15 @@ func main() {
 		apiroots[i] = path.Join(root, appname, apiversion)
 	}
 
-	srv, err := NewServer(addr, apiroots, db)
+	msgChannel := make(chan []byte, 100)
+	producer := &ProducerConfig{
+		Brokers: []string{kafkaBootstrap},
+		Topic:   metricsTopic,
+		Async:   true,
+	}
+	go Producer(producer, msgChannel)
```

Comment:
What happens if I call `Producer` not in a go-routine? Will my thread be blocked by the channel for-loop? If that is the case, then I think we should move this `go` call inside the `Producer` function, ensuring that it will always be run inside a goroutine and let the calling thread execution proceed as normal.

Reply:
that's exactly what happens. I'll look at putting it in the Producer